cloudinstall/task.py

```diff
@@ -134,19 +136,28 @@ def update_progress(self, loop=None, userdata=None):
                                               ts='   -')))
             elif e is None:
                 e = time.time()
-                ts = "{:6.2f} sec elapsed".format(e - s)
+                tdelta = e - s
+                ts = "{:6} sec(s) elapsed".format(
+                    datetime.timedelta(seconds=int(tdelta)).seconds)
                 m.append("{n:>{mw}}: {ts:<22}"
                          "\n".format(n=n, mw=self.max_width, ts=ts))
                 if self.task_info_func:
                     m.append(('label',
                               "\n{}\n\n".format(self.task_info_func())))
             else:
-                ts = "{:6.2f} sec".format(e - s)
+                tdelta = e - s
+                ts = "{:6} sec(s) elapsed".format(
+                    datetime.timedelta(seconds=int(tdelta)).seconds)
```

Comment:
I left 'elapsed' off the completed tasks because I thought it made the current one stick out more.
I'm not sure I care enough to argue for keeping it that way, but thought I might as well explain that it was intentional.